CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- `Client::current_info` and `FaktoryState` struct
+- `Client` method for pausing, resuming, and removing queues
+- TLS configurations options to `WorkerBuilder`
+- `Error::Stream` for underlying 'native_tls' and 'rustls' errors
+- Shutdown signal via `WorkerBuilder::with_graceful_shutdown`
+- Shutdown timeout via `WorkerBuilder::shutdown_timeout`
+- [Faktory Enterprise Edition] Batch jobs (`Batch`, `BatchId`, `BatchStatus`)
+- [Faktory Enterprise Edition] Setting and getting a job's progress
```

Comment:
You'll need a link target for this defined somewhere

Reply:
I actually did not mean this to be a hyperlink, maybe a colon would be better